src/main/java/com/alkemy/ong/service/impl/UserServiceImpl.java

```diff
@@ -61,6 +65,12 @@ public UserResponseDto saveUser(RegisterRequest registerRequest) {
 
         newUser.setPassword(passwordEncoder.encode(newUser.getPassword()));
 
+        Role roleUser = rolesRepository.findById(2L).orElseThrow(() -> {
+            throw new NotFoundException("Role not found.");
```

Comment:
Maybe it is a good idea to log the error and throw an internal server error since this should never happen.